fullstack-cert/python-projects/lab-confguration-manager/main.py

```diff
@@ -0,0 +1,71 @@
+def add_setting(settings):
+    key = input("Enter setting name to add: ").lower()
```

Comment:
are we able to test this? doesn't this waiting for an input block execution?

Reply:
Maybe we should remove the `input` calls and pass in a value as a second argument. This would remove the interactive menu but it would be easier to test everything.